src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/ThreadStatics.cs

```diff
@@ -17,11 +17,72 @@ namespace Internal.Runtime
     /// </summary>
     internal static class ThreadStatics
     {
+        private static int s_inlineTlsLength;
+        private static unsafe int GetInlineTlsLength(TypeManagerHandle typeManager)
+        {
```

Comment:
This is essentially returning number of types that contains TLS fields, right? If so, can we rename this to something that captures the intent instead of `TlsLength` part? Also, may be a method summary would be helpful.